tests/unit_tests/c_wrappers/integration/test_parameter_sample_types.py

```diff
@@ -306,15 +306,56 @@ def test_surface_param(
         # Assert that the data has been internalised to storage
         fs = ert.getEnkfFsManager().getFileSystem("default")
         if expect_num_loaded > 0:
-            arr = fs.load_parameter(
-                ert.ensembleConfig(), [0], update.Parameter("MY_PARAM")
-            )
+            arr = fs.load_surface_data("MY_PARAM", [0])
             assert arr.flatten().tolist() == [0.0, 1.0, 2.0, 3.0]
         else:
             with pytest.raises(KeyError, match="No parameter: MY_PARAM in storage"):
-                fs.load_parameter(
-                    ert.ensembleConfig(), [0], update.Parameter("MY_PARAM")
-                )
+                fs.load_surface_data("MY_PARAM", [0])
+
+
+@pytest.mark.parametrize(
+    "config_str",
+    [
+        "SURFACE MY_PARAM OUTPUT_FILE:surf.irap   INIT_FILES:surf.irap   BASE_SURFACE:surf0.irap",  # noqa
+    ],
+)
+def test_copy_case(
+    tmpdir,
+    config_str,
+):
+    with tmpdir.as_cwd():
+        config = dedent(
+            """
+        JOBNAME my_name%d
+        NUM_REALIZATIONS 10
+        """
+        )
+        config += config_str
+        expect_surface = Surface(
+            nx=2, ny=2, xinc=1, yinc=1, xstart=1, ystart=1, angle=0
+        )
+        for i in range(4):
+            expect_surface[i] = float(i)
+        expect_surface.write("surf.irap")
+        expect_surface.write("surf0.irap")
+
+        with open("config.ert", "w") as fh:
+            fh.writelines(config)
+        ert = create_runpath("config.ert", active_mask=[True for _ in range(10)])
+
+        # Assert that the data has been internalised to storage
+        fs = ert.getEnkfFsManager().getFileSystem("default")
+        new_fs = ert.getEnkfFsManager().getFileSystem("copy")
```

Comment:
`getEnkfFsManger` is only returning `self`, use `ert.storage_manager["default"] instead, if you need to create a case it is:
`ert.storage_manager.add_case("new_case")`